### DISCUSSION TITLE
DynamoDb Support Contribution

### DISCUSSION THREAD

**Discussion by kelvindesman** [fastapi-users/fastapi-users#1153] [Ideas]
Hello!

I love the `fast-API-users` package and using it for production.

I've done some research and don't found any out-of-the-box support for DynamoDB, there is some interesting discussion [here](https://github.com/fastapi-users/fastapi-users/issues/271) but it seems not maintained anymore. 

I created my own DB adapter based on the `fastapi-users-db-sqlalchemy`. I'd like to know how to officially share my project so it became part of the official adapter and can be used by other dev.

**Comment by frankie567** [comment]
Hi @vousmeevoyez 👋 

That's great to here! Well, if you publish it on PyPi and share the GitHub repository, I would be happy to list it in the docs as a "community-supported" adapter 😄
